app/presenters/chf_index_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class ChfIndexPresenter < Blacklight::IndexPresenter
+  class_attribute :description_max_chars
+  self.description_max_chars = 10
+
+  attr_reader :description_field
+
+  def initialize(*args)
+    super
+    # This is an attempt to not hard-code in that our field is called
+    # `description_tesim`, so that doesn't break if it's changed, we
+    # just look for the "description" field. Not really sure if this is
+    # really less fragile, or if there was a better place to get this.
+    @description_field = configuration.index_fields.values.
+      find { |conf| conf.itemprop == "description" }.
```

Comment:
I was trying to avoid yet another place that has a `solr_name` call that has to match exactly the calls in the two other existing place, all of which have to be changed and kept in sync every time one changes. I am not happy with the hydra architecture here, but this is an attempt to avoid that.